### PR TITLE
Added exports for ContextualMenu getItemStyles and getItemClassNames to v8

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
To better support removing deep imports in ODSP.

## Changes
- added top-level export of ContextualMenu getItemStyles as getContextualMenuItemStyles
- added top-level export of ContextualMenu getItemClassNames as getContextualMenuItemClassNames

## Issues
Updates #24454

To be cherry picked to 7.0 branch.

### CODE REVIEW

File: packages/react/etc/react.api.md [hunk 1]
```diff
@@ -1719,6 +1719,14 @@ export function getColorFromRGBA(rgba: IRGB): IColor;
 // @public
 export function getColorFromString(inputColor: string): IColor | undefined;
 
+// Warning: (ae-internal-missing-underscore) The name "getContextualMenuItemClassNames" should be prefixed with an underscore because the declaration is marked as @internal
```
Comment (spmonahan) [inline packages/react/etc/react.api.md:1722]: This warning is probably worth looking into more but I don't think it should block this.

**Review by spmonahan — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6d33373dc12d4595e332d7c39e4fdec3f3081e24'>6d33373dc12d4595e332d7c39e4fdec3f3081e24</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react/etc/react.api.md (hunk 1 above)
Comment (GeoffCoxMSFT) [reply packages/react/etc/react.api.md:1722]: Asked Elizabeth.  It is a just a warning to not use it, but folks to anyway.

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/BaseButton-mount_base.html">876</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/BaseButton-mount.html">893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Breadcrumb-mount_base.html">2838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Breadcrumb-mount.html">2810</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Checkbox-mount_base.html">2583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Checkbox-mount.html">2583</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/CheckboxBase-mount_base.html">2256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/CheckboxBase-mount.html">2273</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ChoiceGroup-mount_base.html">4599</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ChoiceGroup-mount.html">4626</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ComboBox-mount_base.html">971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ComboBox-mount.html">967</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/CommandBar-mount_base.html">10487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/CommandBar-mount.html">10553</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ContextualMenu-mount_base.html">12255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ContextualMenu-mount.html">12108</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DefaultButton-mount_base.html">1121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DefaultButton-mount.html">1126</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DetailsRow-mount_base.html">3739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DetailsRow-mount.html">3855</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DetailsRowFast-mount_base.html">3753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DetailsRowFast-mount.html">3807</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DetailsRowNoStyles-mount_base.html">3628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DetailsRowNoStyles-mount.html">3637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Dialog-mount_base.html">3090</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Dialog-mount.html">3113</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DocumentCardTitle-mount_base.html">185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/DocumentCardTitle-mount.html">173</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Dropdown-mount_base.html">3275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Dropdown-mount.html">3317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/FocusTrapZone-mount_base.html">1860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/FocusTrapZone-mount.html">1881</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/FocusZone-mount_base.html">1772</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/FocusZone-mount.html">1781</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/IconButton-mount_base.html">1775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/IconButton-mount.html">1709</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Label-mount_base.html">362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Label-mount.html">365</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Layer-mount_base.html">4436</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Layer-mount.html">4416</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Link-mount_base.html">461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Link-mount.html">470</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/MenuButton-mount_base.html">1467</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/MenuButton-mount.html">1463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/MessageBar-mount_base.html">2131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/MessageBar-mount.html">2198</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Nav-mount_base.html">3367</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Nav-mount.html">3344</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/OverflowSet-mount_base.html">1085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/OverflowSet-mount.html">1116</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Panel-mount_base.html">2492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Panel-mount.html">2472</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Persona-mount_base.html">999</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Persona-mount.html">976</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Pivot-mount_base.html">1422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Pivot-mount.html">1554</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/PrimaryButton-mount_base.html">1310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/PrimaryButton-mount.html">1272</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Rating-mount_base.html">7826</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Rating-mount.html">7636</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SearchBox-mount_base.html">1280</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SearchBox-mount.html">1312</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Shimmer-mount_base.html">2875</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Shimmer-mount.html">2887</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Slider-mount_base.html">1919</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Slider-mount.html">1975</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SpinButton-mount_base.html">5000</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SpinButton-mount.html">4927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Spinner-mount_base.html">423</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Spinner-mount.html">421</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SplitButton-mount_base.html">3120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SplitButton-mount.html">3146</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Stack-mount_base.html">503</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Stack-mount.html">520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/StackWithIntrinsicChildren-mount_base.html">2229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/StackWithIntrinsicChildren-mount.html">2194</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/StackWithTextChildren-mount_base.html">5125</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/StackWithTextChildren-mount.html">5113</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SwatchColorPicker-mount_base.html">11664</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/SwatchColorPicker-mount.html">11782</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/TagPicker-mount_base.html">2613</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/TagPicker-mount.html">2603</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/TeachingBubble-mount_base.html">95570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/TeachingBubble-mount.html">95261</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Text-mount_base.html">425</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Text-mount.html">426</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/TextField-mount_base.html">1358</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/TextField-mount.html">1378</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ThemeProvider-mount_base.html">1314</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ThemeProvider-mount.html">1305</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ThemeProvider-virtual-rerender_base.html">762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ThemeProvider-virtual-rerender.html">769</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2057</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Toggle-mount_base.html">818</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/Toggle-mount.html">826</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/buttonNative-mount_base.html">137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24484/perf-test/buttonNative-mount.html">130</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**